README.rst

```diff
@@ -100,6 +100,11 @@ Then, launch the front-end development server by running the following command:
 
     $ yarn start
 
+If you want to use a test database and not the backend, you can run on another terminal
```

Comment:
I would change this sentence to something more explicit.

```suggestion
If you want to launch the front-end test server (without launching the backend) with dummy data, execute:
```

Reply:
:ok_hand: